eng/source-build-patches/0001-Remove-test-projects-from-solution.patch

```diff
@@ -0,0 +1,148 @@
+From 194c172bb1d8e0c0fa0596062ccdd3c28a3923b4 Mon Sep 17 00:00:00 2001
```

Comment:
There are ways how to do nicely. Could you do it without patching sln?

Reply:
Any suggestions how? Looking at `build.sh`, it directly kicks off `illnk.sln`. I dont see too many options for conditionalization. Are you thinking of adding a separate sln file that we can use in a source-build context?

Reply:
we could add a build.proj file to do the actual build instead of building the .sln (and keep the .sln just for the IDE).

Reply:
what Alexander said or add another configuration (e.g. Debug, Release, SourceBuild)to sln which does not include tests

Reply:
Done, I think.